mmdeploy/pytorch/symbolics/squeeze.py

```diff
@@ -18,4 +20,10 @@ def squeeze__default(g, self, dim=None):
                 dims.append(i)
     else:
         dims = [sym_help._get_const(dim, 'i', 'dim')]
+
+    ctx = SYMBOLIC_REWRITER.get_context('squeeze')
+    if get_ir_config(ctx.cfg)['opset_version'] >= 13:
```

Comment:
opset_version can be get from [g.opset](https://github.com/pytorch/pytorch/blob/6a866c3ed180c05520e2114e48e97bf1e8b2e1a3/torch/onnx/symbolic_helper.py#L815). So we can export it without deploy config.

Reply:
I tried to but failed because our `g` is an object of `torch._C.graph` instead of `jit_utils.GraphContext`.